cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/kv/HBaseKVTableTest.java

```diff
@@ -48,10 +50,15 @@
   public static void beforeClass() throws Exception {
     testHBase = new HBaseTestFactory().get();
     testHBase.startHBase();
+    hBaseTableUtil.createNamespaceIfNotExists(testHBase.getHBaseAdmin(), NAMESPACE_ID);
   }
 
   @AfterClass
   public static void afterClass() throws Exception {
+    Pattern cdapTablesPattern = Pattern.compile("cdap_" + NAMESPACE_ID.getId() + ".*");
+    testHBase.getHBaseAdmin().disableTables(cdapTablesPattern);
+    testHBase.getHBaseAdmin().deleteTables(cdapTablesPattern);
+    hBaseTableUtil.deleteNamespaceIfExists(testHBase.getHBaseAdmin(), NAMESPACE_ID);
```

Comment:
I thought our table names use '.' not '_' or did we change that?  Also, can we extract this to some common place instead of in every test class.